README.md

```diff
@@ -239,15 +239,15 @@ Valet guarantees that reading and writing operations will succeed as long as wri
 
 ## Requirements
 
-* Xcode 13.0 or later.
-* iOS 9 or later.
-* tvOS 9 or later.
-* watchOS 2 or later.
-* macOS 10.11 or later.
+* Xcode 16.0 or later.
+* iOS 12 or later.
+* tvOS 12 or later.
+* watchOS 4 or later.
+* macOS 10.13 or later.
 
 ## Migrating from prior Valet versions
 
-The good news: most Valet configurations do _not_ have to migrate keychain data when upgrading from an older version of Valet. All Valet objects are backwards compatible with their counterparts from prior versions. We have exhaustive unit tests to prove it (search for `test_backwardsCompatibility`). Valets that have had their configurations deprecated by Apple will need to migrate stored data.
```

Comment:
the exhaustive tests required the v2 (objc) version of Valet to live forever without changes, and these old files were utilizing deprecated API. We maintained these tests for the better part of a decade. We successfully managed to deploy a rewrite in Swift without regression years ago. Time to let go.